android/build.gradle

```diff
@@ -91,34 +91,34 @@ dependencies {
   // Java concurrency annotations
   implementation 'net.jcip:jcip-annotations:1.0'
 
-  implementation 'androidx.constraintlayout:constraintlayout:2.0.4'
+  implementation 'androidx.constraintlayout:constraintlayout:2.1.0'
   implementation 'androidx.work:work-runtime:2.5.0'
   implementation 'com.trafi:anchor-bottom-sheet-behavior:0.13-alpha'
   implementation 'com.github.yoksnod:MPAndroidChart:3.2.0-alpha'
-  implementation 'com.google.android.material:material:1.4.0-beta01'
-  implementation 'androidx.appcompat:appcompat:1.2.0'
+  implementation 'com.google.android.material:material:1.5.0-alpha02'
```

Comment:
Do we really need to use alpha and beta versions of dependencies? Why not to use last stable version?

Reply:
It's ok if it works. Let's stabilize when they release 1.5
In 1.5 branch they added compileSdkVersion 30 as a target and included many other fixes.